src/content/resources/glossary.md

```diff
@@ -279,3 +279,33 @@ If not, the pattern _refutes_, or denies, the match.
 Refutable patterns appear in [_matching contexts_][].
 
 [_matching contexts_]: /language/patterns#matching
+
+## Subclass
+
+A _subclass_ is a class that inherits from another class using the
+[`extends`](/language/extend) keyword.
+
+```dart
+class A extends B {} // A is a subclass of B; B is the superclass of A. 
+```
+
+Any subclass relation is also a [subtype](#subtype) relation,
+so you can use "subtype" to describe any subclass in Dart.
+However, the inverse is not true. Not all subtypes are subclasses.
+See the [subtype](#subtype) entry for more information.
+
+## Subtype
+
+A _subtype_ is a class that implements another class
```

Comment:
This isn't strictly true as a definition of "subtype".  There are types which are not classes.  If you want to restrict yourself to classes here, you might say something like "a class is a subtype of another class if it implements or inherits from it (directly or indirectly)".